validation-test/stdlib/UnsafeBufferPointer.swift.gyb

```diff
@@ -15,6 +15,8 @@ struct SubscriptGetTest {
   /// The values that should be expected by slicing the UBP, or `nil` if the
   /// test is expected to crash.
   let expectedValues: [Int]?
+  /// Same as `expectedValues`, but for closed ranges. `nil` if no difference.
+  let expectedClosedValues: [Int]?
```

Comment:
There is `let subscriptRangeTests` in StdlibCollectionUnittest, which is very similar in intent.